index.html

```diff
@@ -122,6 +177,196 @@ <h2>Methods</h2>
+                  a mandatory-to-implement extension, throw an IllegalArgument error.
+                </li>
+                <li>
+                  Set the "direction" field of <var>entry</var> to <var>ext</var>'s
+                  "direction" field.
+                </li>
+              </ol>
+            </li>
+          </ol>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h2>Attributes</h2>
+      <dl>
+        <dt>headerExtensionsToOffer, readonly</dt>
+        <dd>Returns the value of the [[\HeaderExtensionsToOffer]] slot.</dd>
+      <dl>
+        <dt>headerExtensionsNegotiated, readonly</dt>
+        <dd>Returns the value of the [[\HeaderExtensionsNegotiated]] slot.</dd>
+      </dl>
+    </section>
+    <section>
+      <h2>Modifications to existing procedures</h2>
+      <p>
+        In the <a>final steps to create an offer</a>, add a step:
+      </p>
+      <ol>
+        <li>If the [[\HeaderExtensionsToOffer]] internal slot is set, for
+          each extension in [[\HeaderExtensionsToOffer]], add an appropriate
+          "a=hdrext" line.
+        </li>
+      </ol>
+      <p>
+        In the <a>set an RTCSessionDescription</a> algorithm, add a step
+        right after the step that sets transceiver.[[\Sender]].[[\SendCodecs]],
+        saying "For each transciever, set [[\HeaderExtensionsNegotiated]] to
+        an empty list, and then
+        for each "a=hdrext" line, add an appropriate
+        RTCRtpHeaderExtensionCapability to the list.
+      </p>
+    </section>
+  </section>
+  <section id="rtcrtpheaderextensionparameters-interface">
+    <h3>
+      <code><a>RTCRtpHeaderExtensionParameters</a></code> extensions
+    </h3>
+    <p>
+      The <code><a>RTCRtpHeaderExtensionParameters</a></code>
+      dictionary is defined in [[WEBRTC]]. This document extends that interface
+      by adding an additional member.
+    </p>
+    <pre class="idl">
+      partial dictionary RTCRtpHeaderExtensionParameters {
+        boolean enabled;
+      };
+    </pre>
+    <section>
+      <h2>Attributes</h2>
+      <dl>
+        <dt>enabled, of type boolean</dt>
+        <dd>
+          <p>When returned from getParameters() on an RTPSender, "enabled" is true
+            when the RTP sender is
+            configured to send this header extension when appropriate. If the
+            attribute is missing, it means that it is enabled and cannot be disabled.
+          </p>
+          <p>
+            When passed to setParameters(), setting "enabled" to true instructs the
+            RTP sender to send this header extension when appropriate. Setting "enabled"
+            to false instructs the RTP sender to never send this extension.
+          </p>
+          <p>
+            When calling getParameters() on an RTPReceiver, the "enabled" member will
+            always be missing, indicating that there is no control surface here.
```

Comment:
nit: "indicating that there is no control surface here" is not needed, because RTCRtpReceiver.setParameters() is not a method that exists, so this is true for anything that it returns. Just say it's always missing.

Reply:
Done.